### PR TITLE
feat(bootstrap): custom gtag.js url

### LINKED ISSUE #167: [Feature Request] custom gtag.js url
Hey all 👋🏻

The predecessor plugin, `vue-analytics` had the feature to provide a [custom analytics URL](https://matteogabriele.gitbooks.io/vue-analytics/docs/custom-analytics.html). With `vue-gtag`, this doesn't seem possible at the moment.

Would you accept a PR implementing such a feature?

### PR DESCRIPTION
**Pull request by Yihao-G**
Allow users to customise gtag.js URL.

Fixes #167

### CODE REVIEW

File: src/bootstrap.js [hunk 1]
```diff
@@ -45,8 +46,8 @@ export default function () {
     return;
   }
 
-  const domain = "https://www.googletagmanager.com";
-  const resource = `${domain}/gtag/js?id=${config.id}&l=${globalDataLayerName}`;
+  const resource = `${customResourceURL}?id=${config.id}&l=${globalDataLayerName}`;
+  const domain = new URL(customResourceURL).origin;
```
Comment (MatteoGabriele) [inline src/bootstrap.js:50]: unfortunately the URL api is not supported by IE

**Review by MatteoGabriele — CHANGES_REQUESTED** [review]
thanks for the contribution

**Comment by Yihao-G** [comment]
Finally, so many strict checks and linters 😂

**Comment by MatteoGabriele** [comment]
wow. thanks for your work but it seems like this `getUrlOrigin` is almost as long as the entire plugin itself ahahahahha
what about just going with a default domain value, as the gtag one, and then make it available for users to change it accordingly?

**Comment by Yihao-G** [comment]
How about this one now?

**Review by MatteoGabriele — APPROVED** [review]
looking good! thanks a lot for sorting this out!

**Pull request merged**